### PR TITLE
[109] add ismodifed on tracking actions

### PR DESCRIPTION
**Pull request by mateuszpmroz**

### CODE REVIEW

File: src/react/components/pages/editorPage/editorPage.tsx [hunk 1]
```diff
@@ -259,96 +287,110 @@ export default class EditorPage extends React.Component<IEditorPageProps, IEdito
...
-        }, () => this.canvas.current.applyTag(tag.name));
-    }
+        this.setState(
+            {
+                selectedTag: tag.name,
+                lockedTags: []
+            },
+            () => this.canvas.current.applyTag(tag.name)
+        );
+    };
```
Comment (mateuszpmroz) [inline src/react/components/pages/editorPage/editorPage.tsx]: @bklukaczewski In most of the time husky is working great, but in this case I think that previous version was better :thinking:

Comment (mateuszpmroz) [reply src/react/components/pages/editorPage/editorPage.tsx]: @johannamelly What do you think?

Comment (bklukaczewski) [reply src/react/components/pages/editorPage/editorPage.tsx]: Hard to say what you're referring to ;) 
Are there any issues with pre-commit? Is formatting broken?

Comment (mateuszpmroz) [reply src/react/components/pages/editorPage/editorPage.tsx]: @bklukaczewski  No, everything is fine I was reffering to the `setState`. The formating is changing sometimes, but sometimes not. But that's not something that's really important :p 

```
        this.setState({
            selectedTag: tag.name,
            lockedTags: CanvasHelpers.toggleTag(locked, tag.name),
        }, () => this.canvas.current.applyTag(tag.name));


        this.setState(
            {
                selectedTag: tag.name,
                lockedTags: CanvasHelpers.toggleTag(locked, tag.name)
            },
            () => this.canvas.current.applyTag(tag.name)
        );
   
```

**Pull request merged**